### PR TITLE
Issue #249 - Privacy policy

### PR DESCRIPTION
**Pull request by kubili**
#249

### CODE REVIEW

File: piplmesh/frontend/templates/privacy.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% extends "plain.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Privacy policy" %}{% endblock %}
+
+{% block content %}
+    <div class="Privacy">
```
Comment (mitar) [inline piplmesh/frontend/templates/privacy.html]: Why this div?

File: piplmesh/frontend/templates/privacy.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <h1>{% trans "Privacy policy" %}</h1>
+        <p>{% blocktrans %}<em>Last modified: Septembr 6, 2012</em>{% endblocktrans %}</p>
```
Comment (mitar) [inline piplmesh/frontend/templates/privacy.html]: `<em>` could be outside of block trans.

File: piplmesh/frontend/templates/privacy.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        <h2>{% trans "Personal identification information" %}</h2>
+        <p>{% blocktrans %}We may collect personal identification information from users in a variety of ways, including, but not limited to, when users visit our site or register on the site. Users may be asked for, as appropriate, name and/or email address. Users may, however, visit our site anonymously. We will collect personal identification information from users only if they voluntarily submit such information to us. Users can always refuse to supply personally identification information.{% endblocktrans %}</p>
+        <p>{% blocktrans %}We will never ask you for your password or any other personal information via email. Never reply to such an email. If you have any questions regarding the validity of an email sent to you, contact us via support system or web page.{% endblocktrans %}</p>
```
Comment (mitar) [inline piplmesh/frontend/templates/privacy.html]: I think we are everywhere writing `e-mail` and not `email`. But please check.

Comment (mitar) [inline piplmesh/frontend/templates/privacy.html]: You can provide here link to our contact page. And also change "support system or web page" text accordingly.

**Comment by mitar** [comment]
Check if this code is XHTML correct. You can use [HTML Validator](http://users.skynet.be/mgueury/mozilla/) for this.

File: piplmesh/frontend/templates/privacy.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        <h2>{% trans "Non-personal identification information" %}</h2>
+        <p>{% blocktrans %}We may collect non-personal identification information about users type of device, the browser name, the operating system and other similar technical information. This includes the users location as it is connected to the network via access point whoose location is already known.{% endblocktrans %}</p>
+        <h2>{% trans "Web browser cookies" %}</h2>
+        <p>{% blocktrans %}Our site may use "cookies" to enhance your experience. Users web browser places cookies on their hard drive for record-keeping purposes and sometimes to track information about them. User may choose to set their web browser to refuse cookies. If they do so, note that some parts of the site may not function properly.{% endblocktrans %}</p>
+        <h2>{% trans "How we use information we collected" %}</h2>
+        <p>{% blocktrans %}We may collect and use users personal information to:
+        <ul><li>improve customer service (helps us respond to your customer service requests and support),</li>
```
Comment (mitar) [inline piplmesh/frontend/templates/privacy.html]: Rewrap/indent better (see how we do it elsewhere. Check XHTML compliance.

Don't have newlines in blocktrans.

**Comment by mitar** [comment]
It sounds good. I am not sure if some parts are not maybe parts of terms of use instead of privacy policy? For example, things about external links?

Have you checked Google and Facebook privacy policy? Are there anything you missed?

**Comment by kubili** [comment]
I checked and updated it.

File: piplmesh/frontend/templates/privacy.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+{% block title %}{% trans "Privacy policy" %}{% endblock %}
+
+{% block content %}
+    <h1>{% trans "Privacy policy" %}</h1>
+    <p><em>{% blocktrans %}Last modified: Septembr 7, 2012{% endblocktrans %}</em></p>
+    <h2>{% trans "Personal identification information" %}</h2>
+    <p>{% blocktrans %}We may collect personal identification information from users in a variety of ways, including, but not limited to, when users visit our site or register on the site. Users may be asked for, as appropriate, name and/or e-mail address. Users may, however, visit our site anonymously. We will collect personal identification information from users only if they voluntarily submit such information to us. Users can always refuse to supply personally identification information.{% endblocktrans %}</p>
+    {% url contact as contact_url %}
+    <p>{% blocktrans %}We will never ask you for your password or any other personal information via e-mail. Never reply to such an e-mail. If you have any questions regarding the validity of an e-mail sent to you, contact us via our <a href="{{ contact_url }}">contact page</a>.{% endblocktrans %}</p>
```
Comment (mitar) [inline piplmesh/frontend/templates/privacy.html]: I would add that they can simply reply to the e-mail or use our contact page.

File: piplmesh/frontend/templates/privacy.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+{% load i18n %}
+
+{% block title %}{% trans "Privacy policy" %}{% endblock %}
+
+{% block content %}
+    <h1>{% trans "Privacy policy" %}</h1>
+    <p><em>{% blocktrans %}Last modified: Septembr 7, 2012{% endblocktrans %}</em></p>
+    <h2>{% trans "Personal identification information" %}</h2>
+    <p>{% blocktrans %}We may collect personal identification information from users in a variety of ways, including, but not limited to, when users visit our site or register on the site. Users may be asked for, as appropriate, name and/or e-mail address. Users may, however, visit our site anonymously. We will collect personal identification information from users only if they voluntarily submit such information to us. Users can always refuse to supply personally identification information.{% endblocktrans %}</p>
+    {% url contact as contact_url %}
```
Comment (mitar) [inline piplmesh/frontend/templates/privacy.html]: We are using `url` template tag from the `future`. You should load `future tag library above and then you have to use a different syntax here:

```
{% "url" contact as contact_url %}
```

File: piplmesh/frontend/templates/privacy.html [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+        <li>{% blocktrans %}send periodic e-mails (as response to their inquiries, questions, and/or other requests).{% endblocktrans %}</li>
+    </ul>
+    <p>{% blocktrans %}Your personal information may be processed outside the country where you live.{% endblocktrans %}</p>
+    <h2>{% trans "What can you do with collected information" %}</h2>
+    <ul>
+        <li>{% blocktrans %}Review and control your information,{% endblocktrans %}</li>
+        <li>{% blocktrans %}Update information or to delete it, unless we have to keep it for legal purposes,{% endblocktrans %}</li>
+        <li>{% blocktrans %}Export your account so all of your informations linket to it will be exported,{% endblocktrans %}</li>
+        <li>{% blocktrans %}Disable your account so all of your informations become hidden or{% endblocktrans %}</li>
+        <li>{% blocktrans %}Delete your account so all of your informations linket to it will be also deleted{% endblocktrans %}</li>
```
Comment (mitar) [inline piplmesh/frontend/templates/privacy.html:33]: Dot missing at the end.

Comment (mitar) [inline piplmesh/frontend/templates/privacy.html:33]: "linket" -> "linked"

**Pull request merged**